src/Stryker.Core/Stryker.Core/Logging/ApplicationLogging.cs

```diff
@@ -5,23 +5,36 @@
 
 namespace Stryker.Core.Logging
 {
+    using LibGit2Sharp;
```

Comment:
Why is this placed inside of the namespace? Every file in our project uses logging and thus uses LibGit2Sharp now? I don't think thats needed.